### PR TITLE
adds wheelchair to utility loadout

### PR DESCRIPTION
**Pull request by spessbro**
## About The Pull Request
As of right now you can remove your legs by amputation at start but can't start in a wheelchair so I think it's nice to have it able to get it from the get go instead of crawling until you make/find one and makes sense lore-wise since poor vagabonds might not be able to afford prosthetics while still getting badly wounded on the regular.

### CODE REVIEW

**Comment by spessbro** [comment]
what the frick travis :(

File: code/modules/client/preference_setup/loadout/lists/utility.dm [hunk 1]
```diff
@@ -37,6 +37,11 @@
 	path = /obj/item/modular_computer/tablet/preset/custom_loadout/cheap
 	cost = 2
 
+/obj/structure/bed/chair/wheelchair
+	display_name = "wheelchair"
+	pathc = /obj/structure/bed/chair/wheelchair
```
Comment (TorinoFermic) [inline code/modules/client/preference_setup/loadout/lists/utility.dm]: /obj/structure/bed/chair/wheelchair points to  /obj/structure/bed/chair/wheelchair, well it won't compile.
You misspelled pathc for path

Comment (spessbro) [reply code/modules/client/preference_setup/loadout/lists/utility.dm]: thank you i am very dumb and can only webedit on linux

**Comment by MadmanMartian** [comment]
A lot of loadout code expects the item to be, well, an item.

You may want to comb through loadout code and see where it could potentially runtime

**Comment by APWill** [comment]
> A lot of loadout code expects the item to be, well, an item.
> 
> You may want to comb through loadout code and see where it could potentially runtime

Since I implemented the foldable wheelchair functionality (kindly edited by SolatK) it's already available in item form as the folded version. Just change the path to /obj/item/wheelchair.

File: code/modules/client/preference_setup/loadout/lists/utility.dm [hunk 2]
```diff
@@ -36,6 +36,10 @@
 	display_name = "cheap tablet computer"
 	path = /obj/item/modular_computer/tablet/preset/custom_loadout/cheap
 	cost = 2
+/datum/gear/utility/wheelchair
+	display_name = "wheelchair"
+	path = /obj/structure/bed/chair/wheelchair
```
Comment (MadmanMartian) [inline code/modules/client/preference_setup/loadout/lists/utility.dm]: Replace with the foldable wheelchair object

Comment (TorinoFermic) [reply code/modules/client/preference_setup/loadout/lists/utility.dm]: The path must be a item

**Pull request merged**